staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -21,13 +21,22 @@ import (
 	"strconv"
 	"strings"
 
-	"k8s.io/apimachinery/pkg/apimachinery/registered"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	serverstore "k8s.io/apiserver/pkg/server/storage"
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 )
 
+// Interface provides access to registered group versions.
+type Interface interface {
```

Comment:
GroupRegistry? Interface is not a good name for a generically called package "resourceconfig". This is not the interface abstracting a resource config.

Reply:
Maybe GroupVersionRegistry is better.

Reply:
done